lib/node_modules/@stdlib/stats/base/dists/uniform/median/benchmark/benchmark.js

```diff
@@ -1,7 +1,7 @@
 /**
 * @license Apache-2.0
 *
-* Copyright (c) 2018 The Stdlib Authors.
+* Copyright (c) 2024 The Stdlib Authors.
```

Comment:
We don't update copyright years when changing existing files.

```suggestion
* Copyright (c) 2018 The Stdlib Authors.
```